package.json

```
@@ -46,7 +47,8 @@
     "format-check": "prettier --check '**/*.{ts,tsx,js}'",
     "jest": "jest",
     "test": "yarn run jest && yarn run format-check && yarn run tslint",
-    "tslint": "tslint --project ./tsconfig.json"
+    "tslint": "tslint --project ./tsconfig.json",
+    "serve": "npm run build && clear && gatsby serve"
```

```suggestion
    "serve": "yarn build && clear && gatsby serve"
```

* Use `yarn` consistently instead of ~`npm`~ since the other scripts are using `yarn`?
* `yarn` script can omit the ~"run"~ keyword.

---

Oh that makes perfect sense. I am a heavy `npm` user so missed it.

---

Fixed via eeb0129